src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const tfoot = table.querySelector('tfoot');
+const thElements = document.querySelectorAll('table thead th');
+const positionTh = thElements[1];
+const tfootThElements = document.querySelectorAll('table tfoot th');
+const tfootTh = tfootThElements[1];
```

Comment:
Similar to the previous comment, ensure that the second 'th' element exists in the 'tfoot'. This will prevent potential 'undefined' errors if the table structure is not as expected.